src/civis/parallel.py

```diff
@@ -8,45 +8,16 @@
 import pickle  # nosec
 from tempfile import TemporaryDirectory
 import time
-import warnings
 
 import cloudpickle
-from joblib._parallel_backends import ParallelBackendBase
-
-with warnings.catch_warnings():
-    warnings.simplefilter("ignore", DeprecationWarning)
-    from joblib.my_exceptions import TransportableException
-
-from joblib import register_parallel_backend as _joblib_reg_para_backend
+from joblib.parallel import ParallelBackendBase
+from joblib import register_parallel_backend
 import requests
 
 import civis
 from civis.base import CivisAPIError
 from civis.futures import _ContainerShellExecutor, CustomScriptExecutor
 
-try:
-    with warnings.catch_warnings():
-        # Ignore the warning: "DeprecationWarning: sklearn.externals.joblib is
-        # deprecated in 0.21 and will be removed in 0.23. Please import this
-        # functionality directly from joblib, which can be installed with:
-        # pip install joblib. If this warning is raised when loading pickled
-        # models, you may need to re-serialize those models with
-        # scikit-learn 0.21+."
-        warnings.simplefilter("ignore", DeprecationWarning)
-        # sklearn 0.22 has switched from DeprecationWarning to FutureWarning
-        warnings.simplefilter("ignore", FutureWarning)
-        from sklearn.externals.joblib import (
-            register_parallel_backend as _sklearn_reg_para_backend,
-        )
```

Comment:
[scikit-learn did indeed remove its vendored copy of joblib at v0.23](https://github.com/scikit-learn/scikit-learn/pull/16814). Since its current version is 1.5.0, this hack here is far enough in the past that it should now be safe to drop it for good.